lib/player.js

```diff
@@ -2760,6 +2761,10 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       const onVideoProgress = () => this.onVideoProgress_();
       this.loadEventManager_.listen(
           mediaElement, 'timeupdate', onVideoProgress);
+      this.loadEventManager_.listen(
+          mediaElement, 'playing', onVideoProgress);
```

Comment:
Why is it necessary?

Reply:
I added it as event of interest for the video progress logic. But I guess it's arguably redundant, will remove.

Reply:
Done.